src/model/import.ts

```diff
@@ -59,6 +59,9 @@ export class ScannedImport implements Resolvable {
   }
 
   resolve(document: Document): Import|undefined {
+    if (!document.analyzer.canResolveUrl(this.url)) {
```

Comment:
I suppose it depends what we actually want from the Import object.  Right now it is very convenient for me, in `polymer-bundler` for example to test whether an import's url was imported by seeing if an import with that url exists.  However, I could probably can change the logic to see if that import exists AND if it has a document.

The reason I went with this logic was that the existing behavior was that _unloadable_ documents resulted in an `undefined` resolution instead of an Import without a `document` property.  Would it be right to continue having the failure-to-load case resolve with undefined but the unexceptional unloadable case would resolve with an Import having no document?

What would you prefer?  I can roll with either.